pkg/apis/garden/types.go

```diff
@@ -1571,6 +1586,8 @@ const (
 	ShootSystemComponentsHealthy gardencore.ConditionType = "SystemComponentsHealthy"
 	// ShootAPIServerAvailable is a constant for a condition type indicating the api server is available.
 	ShootAPIServerAvailable gardencore.ConditionType = "APIServerAvailable"
+	// ShootScheduled means the shoot is scheduled to a seed by the gardener-scheduler
+	ShootScheduled gardencore.ConditionType = "ShootScheduled"
```

Comment:
Let's not have this condition. The shoot conditions are meant to represent the health/state of the cluster.

Reply:
ok.
Originally I went  for the update of condition with type ShootScheduled to be consistent with kube-scheduler & arguing that a shoot being assigned to a seed is part of the state of the shoot cluster.

https://github.com/kubernetes/api/blob/master/core/v1/types.go#L2403

I ll remove it - so removing all Shoot Status updates and using Events instead, correct?

Reply:
Yes, at least let's start like that. If we think we need the condition in the future we can change it that way.

Reply:
WDYT if we rather have a 'Phase' for a shoot analogous to the Phase in the kubernetes pods? Then we could describe the Shoot scheduling behavior the same way.

Reply:
@rfranzke ?

Reply:
Not sure, I took a look at https://github.com/kubernetes/api/blob/master/core/v1/types.go#L2380-L2397 to see which phases pods can have. Doesn't seem the right fit.
What are we interested in? If the shoot got scheduled or not (this can be determined by looking at `.spec.cloud.seed`) or whether it could not be scheduled because no seed fits (this can be determined by looking at the events).